xds/src/test/java/io/grpc/xds/SharedXdsClientPoolProviderTest.java

```diff
@@ -51,6 +51,7 @@ public class SharedXdsClientPoolProviderTest {
   @Rule
   public final ExpectedException thrown = ExpectedException.none();
   private final Node node = Node.newBuilder().setId("SharedXdsClientPoolProviderTest").build();
+  private static final String DUMMY_TARGET = "dummy";
```

Comment:
We should test at least two targets, and verify, e.g. the second xdsClient is not the same instance as the first one, is ref counted.

Reply:
I see that multiple targets have been covered in csds service test. Still, it would be nice to have multiple targets tested in this UT.